### DISCUSSION TITLE
Can I have renovate only suggest dependency changes that have a high confidence and passing score?

### DISCUSSION THREAD

**Discussion by lox** [renovatebot/renovate#24289] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

Placeholder value, please select the correct response from the dropdown

### Wanted end result.

I'd love to only see dependency updates that have a high confidence!

### What you tried so far.

Reading docs

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
This feature is possible using the `matchConfidence` option in `packageRules`, however the API required to drive it is currently for Mend.io customers only and soon also for approved Open Source projects.

**Reply by lox** [reply]
We are using mend.io, is it for paying customers only?

**Reply by rarkins** [reply]
If you are paying money for Mend.io's commercial SCA offerings then you are a "customer", if you are using the free hosted Renovate app then not

**Reply by lox** [reply]
Got it, makes sense, thanks!

**Reply by lox** [reply]
Will have a look to see if there are any paid offerings for "Startup Enterprise" type companies like ours.
